src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,122 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import task.Task;
+import task.Tasklist;
+import ui.Parser;
+import ui.Ui;
+
+/**
+ * Storage for todo list information
+ *
+ * @author: Wei Yangken
+ */
+public class Storage {
+
+    private String filepath;
+    private Tasklist tasklist;
+
+    /**
+     * Constructs a storage that stores information at filepath
+     * @param filepath Location of stored information
```

Comment:
Leaving an empty line between the description and parameter section could improve readability.